applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -284,7 +309,7 @@ def FinalizeSolutionStep(self):
 
             # Finalize the solver current step
             self._GetSolutionStrategy().FinalizeSolutionStep()
-
+                self._GetAccelerationLimitationUtility().Execute()
```

Comment:
This is a master conflict wrongly resolved. See that the acceleration correction is checked and applied below.